_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,11 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - "Pandas uses other names for data types than Python, for example: “object" for textual data."
+    - "A column in a DataFrame can only have one data type."
+    - "The data type in a DataFrame’s single column can be checked using `dtype`."
+    - "Make conscious decisions about how to manage missing data."
+    - "A DataFrame can be saved to a CSV file using `pandas.to_csv`."
```

Comment:
use either:

> pandas' `to_csv`

or 

> `pd.to_csv`

The former uses official (lowercase) name for pandas package the the former uses currently-used-in-the-lesson short name `pd` (which, I think, we ought to change to `pandas`)

Reply:
I actually like @wrightaprilm's suggestion (DataFrame can be saved to a CSV file using the to_csv function)

Reply:
Don't sound too surprised, haha;)

```suggestion
    - "DataFrame can be saved to a CSV file using the `to_csv` function."
```
This suggestion should address the comment. Feel free to accept if it looks good to you, @orchid00.